packages/dx-react-grid/docs/reference/integrated-grouping.md

```diff
@@ -12,7 +12,7 @@ A plugin that performs local grouping and group expanding/collapsing.
 
 Name | Type | Default | Description
 -----|------|---------|------------
-getColumnIdentity | (columnName: string) => [Identity](#identity) &#124; undefined | | A function that defines a column's grouping identity. See [Grouping](../guides/grouping.md#local-grouping-with-custom-values) for details.
+getColumnIdentity | (columnName: string) => [Identity](#identity) &#124; undefined | | A function that defines a column's grouping identity. See [Grouping](../guides/grouping.md#integrated-grouping-with-custom-values) for details.
```

Comment:
Why do you use integrated here? Maybe it will be good to change Local in guide headings and demos?